extensions/functions_random.yaml

```diff
@@ -0,0 +1,16 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: random
+    description: "Return a value between 0 (inclusive) and 1 (exclusive)"
+    impls:
+      - deterministic: false
+        return: fp64
+      - args:
+          - value:
+              - name: Seed
+                type: any1
```

Comment:
I don't think the any type makes sense here. Any is really focused on a situation where engines can support any type. Most engines would only support one discrete type (say i64 or fp64). In those cases, we should have discrete implementations.

Reply:
Got it. I'll look at what seed types are commonly supported and I'll add separate implementations for each.